### PR TITLE
bootstrapping express

### PR DESCRIPTION
**Pull request by miguelgimenezgimenez**

### CODE REVIEW

**Comment by 2rod** [comment]
👍

File: controllers/raccoonController.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import raccoon from 'raccoon';
+raccoon.connect(6379, '127.0.0.1');
+
+const raccoonController={};
+
+const cb=()=>{}
+
+raccoonController.like=(req,res)=>{
+  raccoon.lked(req.userID, req.ItemId, cb);
```
Comment (arol) [inline controllers/raccoonController.js]: `liked`?

**Comment by arol** [comment]
Please review the requested changes.

File: controllers/raccoonController.js (hunk 1 above)
Comment (miguelgimenezgimenez) [inline controllers/raccoonController.js]: fixed typo

Comment (arol) [inline controllers/raccoonController.js]: Where?

**Comment by arol** [comment]
Hey what happens with this PR? @miguelgimenezgimenez @2rod

**Comment by miguelgimenezgimenez** [comment]
Its solved in featlogin branch

**Pull request closed without merging**